qiskit_braket_provider/providers/braket_backend.py

```diff
@@ -214,6 +215,39 @@ def qubit_properties(
         # TODO: fetch information from device.properties.provider  # pylint: disable=fixme
         raise NotImplementedError
 
+    def queue_depth(self) -> QueueDepthInfo:
```

Comment:
The Qiskit backend interface expects this information in the [BackendStatus](https://github.com/Qiskit/qiskit/blob/main/qiskit/providers/models/backendstatus.py#L27) object. Could we fit this in there?

Reply:
Qiskit providers uses [BackendV2](https://github.com/qiskit-community/qiskit-braket-provider/blob/main/qiskit_braket_provider/providers/braket_backend.py#L32), while the qiskit backend uses [BackendV1](https://github.com/Qiskit/qiskit/blob/main/qiskit/providers/backend.py#L40), so this change would be incompatible.